stm32/Src/main.c

```diff
@@ -169,7 +177,7 @@ void SystemClock_Config(void)
 
   /** Configure the SYSCLKSource, HCLK, PCLK1 and PCLK2 clocks dividers
   */
-  RCC_ClkInitStruct.ClockType = RCC_CLOCKTYPE_HCLK3|RCC_CLOCKTYPE_HCLK
+  RCC_ClkInitStruct.ClockType = RCC_CLOCKTYPE_HCLK3|RCC_HCLK

```

Comment:
I'm concerned about this change in clock configuration.

Reply:
I'll have to correct this.